drivers/sdhc/rcar_mmc.c

```diff
@@ -906,7 +904,10 @@ static int rcar_mmc_change_voltage(struct sdhc_io *host_io,
 		return ret;
 	}
 
-	host_io->signal_voltage = ios->signal_voltage;
+	if (!ret) {
+		host_io->signal_voltage = ios->signal_voltage;
+	}
+
```

Comment:
You also supposed to reconfigure PMIC, as it should set pull-up voltage to 1.8V.

Right now you are maintaining 3.3V on pull-ups, while you already configured 1.8V in SoC... Not sure that SoC will be happy about this :)

Reply:
Investigating...

Reply:
added fixed-regulator nodes and integrate them with eMMC node and rcar-mmc driver
gpio-regulator I'll add in the future during adding SD card node to dts if you don't mind